### DISCUSSION TITLE
Mod下载界面增加该Mod的简介

### DISCUSSION THREAD

**Discussion by gousiter682** [Meloong-Git/PCL#2817] [讨论]
awa

**Comment by gousiter682** [comment]
希望能增加这个，这样就不用每次都点链接了(我知道这样对龙猫可能很难，在此对龙猫说一声谢谢!)

**Comment by MoYuan-CN** [comment]
你应当开 Issue（ https://github.com/Hex-Dragon/PCL2/issues/new/choose ） ...
Discussion 区只是用来交流的
